src/collections/Grid/GridRow.js

```diff
@@ -41,6 +55,7 @@ GridRow._meta = {
   props: {
     color: SUI.COLORS,
     columns: SUI.WIDTHS,
+    divided: ['vertically'],
```

Comment:
It appears my initial bug report was only half correct.  The row only supports `divided` as bool, not `vertically`.

https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/components/grid.css#L1083